### DISCUSSION TITLE
Is there a way to create a default getter for an object data type?

### DISCUSSION THREAD

**Discussion by ShiiRochi** [handlebars-lang/handlebars.js#1785] [Q&A]
Before filing issues, please check the following points first:

- [x] Please don't open issues for security issues. Instead, file a report at https://www.npmjs.com/advisories/report?package=handlebars
- [x] Have a look at https://github.com/wycats/handlebars.js/blob/master/CONTRIBUTING.md
- [x] Read the FAQ at https://github.com/wycats/handlebars.js/blob/master/FAQ.md
- [ ] Use the jsfiddle-template at https://jsfiddle.net/4nbwjaqz/4/ to reproduce problems or bugs

Link to shared example: [shared example](https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%3Cdiv%3E%5Cn%5Ct%7B%7Buser%7D%7D%5Cn%3Cdiv%2F%3E%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20user%3A%20%5B%7B%20title%3A%20'John'%20%7D%5D%5Cn%7D%5Cn%22%2C%22output%22%3A%22%3Cdiv%3E%5Cn%5Ct%5Bobject%20Object%5D%5Cn%3Cdiv%2F%3E%22%2C%22preparationScript%22%3A%22%2F%2F%20Handlebars.registerHelper('loud'%2C%20function(string)%20%7B%5Cn%2F%2F%20%20%20%20return%20string.toUpperCase()%5Cn%2F%2F%20%7D)%3B%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D)

I've just started to familiarize myself with the package, however, this is the most stunning question I cannot find an appropriate solution to.

Is there a way to be able to tell handlebars to get default property from an object by a default path if it (path) is not specified in a template?

### Example:

I wanna achieve the following result.

##### Template:
```html
<div>
    {{user}}
<div/>
```

##### Data:
```json
{
  "user": { "title": "John Somebody" }
}
```

##### Result:
```htnl
<div>
    John Somebody
<div/>
```

### What's going on here?
In case a path is not specified, `Handlebars` will take a "default path" from an object if it is an object, otherwise puts there an empty string.
Is there a way to accomplish that using helpers or some other custom stuff?
Also, I wanna avoid additional keywords in template HTML as much as possible.

**Comment by jaylinski** [comment]
I'm afraid the only way to achieve this is by using a helper like in [this example](https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%3Cdiv%3E%5Cn%20%20%20%20%7B%7Bget%20user%7D%7D%5Cn%3Cdiv%2F%3E%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20%5C%22user%5C%22%3A%20%7B%20%5C%22title%5C%22%3A%20%5C%22John%20Somebody%5C%22%20%7D%5Cn%7D%22%2C%22output%22%3A%22%3Cdiv%3E%5Cn%20%20%20%20John%20Somebody%5Cn%3Cdiv%2F%3E%22%2C%22preparationScript%22%3A%22Handlebars.registerHelper('get'%2C%20function(context%2C%20options)%20%7B%5Cn%20%20%20%20return%20context.title%3B%20%5Cn%7D)%3B%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D).

Related: https://github.com/handlebars-lang/handlebars.js/issues/1435
